packages/zosfiles/src/methods/create/Create.ts

```diff
@@ -134,7 +134,10 @@ export class Create {
         }
     }
 
-    public static async dataSetLike(session: AbstractSession, dataSetName: string, likeDataSetName: string): Promise<IZosFilesResponse> {
+    public static async dataSetLike(session: AbstractSession,
+                                    dataSetName: string,
+                                    likeDataSetName: string,
+                                    options?: Partial<ICreateDataSetOptions>): Promise<IZosFilesResponse> {
         // Required
         ImperativeExpect.toNotBeNullOrUndefined(dataSetName, ZosFilesMessages.missingDatasetName.message);
         ImperativeExpect.toNotBeNullOrUndefined(likeDataSetName, ZosFilesMessages.missingDatasetLikeName.message);
```

Comment:
Only the "like" option is being passed to the REST client: `JSON.stringify({ like: likeDataSetName })`
Other options in the `options` object must also be passed in. Please modify to stringify those options as well. Something like this should work: `{ like: likeDataSetName, ...(options || {}) }`